eruditus/eruditus.py

```diff
@@ -113,6 +113,9 @@ async def create_ctf(self, name: str, live: bool = True) -> Union[dict, None]:
         notes_channel = await guild.create_text_channel(
             name="📝-notes", category=category_channel, overwrites=overwrites
         )
+        bot_cmds_channel = await guild.create_text_channel(
+            name="🤖-bot-cmds", category=category_channel, overwrites=overwrites
+        )
```

Comment:
It's probably better to create this channel before `notes` and `credentials`, so that it's placed just after `general`.
For the permissions, we can't use that `overwrites` dict because it has `send_messages` set to `False`, I suggest to do:
```python
bot_cmds_channel = await guild.create_text_channel(
    name="🤖-bot-cmds",
    category=category_channel,
    overwrites={
        guild.default_role: discord.PermissionOverwrite(read_messages=False),
        role: discord.PermissionOverwrite(read_messages=True),
    }
)
```
You are welcome to suggest something else.